File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
    int start = 0; int end = num_list.length-1;
    while (start < end){
        int tmp; 
        tmp = num_list[start];
        num_list[start++] = num_list[end];
        num_list[end--] = tmp;
    } 
    return num_list;
}
}

File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.go
func solution(num_list []int) []int {
    start := 0
    end := len(num_list) - 1

    for start < end {
        tmp := num_list[start]
        num_list[start] = num_list[end]
        num_list[end] = tmp

        start++  
        end--
    }
    return num_list
}